UI/ChatUI/ChatUI/ChatUI/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Globalization;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+
+namespace ChatUI.Converters
+{
+    /// <summary>
+    /// Copied/adapted from https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Tests/Converters/FromNullableBoolToVisibilityConverter.cs
+    /// This converter will output a visibility based on if a nullable bool is set to true or otherwise.
+    /// 
+    /// VisibilityIfTrue (VisibilityIfTrue) : Determines the visibility value that will be returned if the value is true.
+    /// 
+    /// By default, VisibilityIfTrue is set to visible.
+    /// 
+    /// This converter may be used to display or hide some content based on a nullable boolean value.
+    /// </summary>
+    public class FromNullableBoolToVisibilityConverter : IValueConverter
```

Comment:
you should also be able to bind visibility to a bool directly, without the needs for a converter

Reply:
Was thinking the same until I noticed that this class supports `Nullable<bool>`, not sure it's supported out-of-the box.

BTW please update the file name to match the class name.

Reply:
File renamed.